dotcom-rendering/src/model/badges.ts

```diff
@@ -0,0 +1,238 @@
+	{
+		seriesTag: 'sport/rugby-world-cup-2019',
+		imageSrc: 'badges/rugby-world-cup.svg',
+	},
+	{
+		seriesTag: 'uk-news/series/behind-the-lines',
+		imageSrc: 'badges/behind-the-lines.svg',
+	},
+	{
+		seriesTag: 'cities/series/the-empty-doorway',
+		imageSrc: 'badges/the-empty-doorway.svg',
+	},
+	{
+		seriesTag: 'world/series/yemen-at-war',
+		imageSrc: 'badges/yemen-at-war.svg',
+	},
+	{
+		seriesTag: 'environment/series/the-polluters',
+		imageSrc: 'badges/the-polluters.svg',
+	},
+	{
+		seriesTag: 'politics/series/lost-in-politics',
+		imageSrc: 'badges/lost-in-politics-badge.svg',
+	},
+	{
+		seriesTag: 'world/series/this-is-europe',
+		imageSrc: 'badges/this-is-europe.svg',
+	},
+	{
+		seriesTag: 'world/series/coronavirus-100-days-that-changed-the-world',
+		imageSrc: 'badges/corona-badge.svg',
+	},
+	{
+		seriesTag: 'australia-news/series/the-green-recovery',
+		imageSrc: 'badges/green-recovery.svg',
+	},
+	{
+		seriesTag: 'environment/series/the-green-recovery',
+		imageSrc: 'badges/green-recovery.svg',
+	},
+	{
+		seriesTag: 'culture/series/culture-in-peril',
+		imageSrc: 'badges/culture-badge.svg',
+	},
+	{
+		seriesTag: 'us-news/series/climate-countdown',
+		imageSrc: 'badges/100days.svg',
+	},
+	{
+		seriesTag: 'society/futureofcities',
+		imageSrc: 'badges/futureofcities.svg',
+	},
+	{
+		seriesTag: 'world/series/the-fight-for-hong-kong',
+		imageSrc: 'badges/eohk.svg',
+	},
+	{
+		seriesTag: 'uk-news/series/spy-cops-scandal',
+		imageSrc: 'badges/spy-cops-scandal.svg',
+	},
+	{
+		seriesTag: 'environment/series/the-last-chance',
+		imageSrc: 'badges/the-last-chance.svg',
+	},
+	{
+		seriesTag: 'australia-news/series/dreams-interrupted',
+		imageSrc: 'badges/dreams-interrupted.svg',
+	},
+	{
+		seriesTag: 'media/series/guardian-200',
+		imageSrc: 'badges/anniversary200.svg',
+	},
+	{
+		seriesTag: 'football/euro-2020',
+		imageSrc: 'badges/euro-2020.svg',
+	},
+	{
+		seriesTag: 'sport/olympic-games-2020',
+		imageSrc: 'badges/tokyo-2020.svg',
+	},
+	{
+		seriesTag: 'sport/paralympic-games-2020',
+		imageSrc: 'badges/tokyo-2020.svg',
+	},
+	{
+		seriesTag: 'sport/tokyo-paralympic-games-2020',
+		imageSrc: 'badges/tokyo-2020.svg',
+	},
+	{
+		seriesTag: 'environment/cop26-glasgow-climate-change-conference-2021',
+		imageSrc: 'badges/cop26-badge.svg',
+	},
+	{
+		seriesTag: 'sport/winter-olympics-2022',
+		imageSrc: 'badges/winter-olympics-2022-badge.svg',
+	},
+	{
+		seriesTag: 'australia-news/australian-election-2022',
+		imageSrc: 'badges/australian-election-2022.svg',
+	},
+	{
+		seriesTag: 'tone/newsletter-tone',
+		imageSrc: 'badges/newsletter-badge.svg',
+	},
+	{
+		seriesTag: 'football/women-s-euro-2022',
+		imageSrc: 'badges/womens_euros_2022_badge.svg',
+	},
+	{
+		seriesTag: 'us-news/us-midterm-elections-2022',
+		imageSrc: 'badges/us-midterm-elections-2022.svg',
+	},
+	{
+		seriesTag: 'football/world-cup-2022',
+		imageSrc: 'badges/world-cup-2022.svg',
+	},
+];
+
+/** @todo - Find out if these still need to be hashed */
```

Comment:
these particular ones may not, but may be good to keep as examples of approach for future super secret investigations